### DISCUSSION TITLE
NvEncoder::LoadNvEncApi : Current Driver Version does not support this NvEncodeAPI version

### DISCUSSION THREAD

**Discussion by panxy2013** [alvr-org/ALVR#1597] [Q&A] [answered]
When i build v19.1.0, meet the following error:
https://github.com/alvr-org/ALVR/releases/tag/v19.1.0

16:43:55.113042500 | [INFO] | Seconds from Vsync to Photons: 0.005000
16:43:55.113113600 | [INFO] | Refresh Rate: 72
16:43:55.114235700 | [INFO] | **### **[GPU PRIO FIX] Failed to set process (-1) priority class, please run ALVR as Administrator.****
16:43:55.133458500 | [ERROR] | Your GPU does not meet the requirements for video encoding. If you get this error after changing some settings, you can revert them by deleting the file "session.json" in the installation folder. Failed to initialize CEncoder: All VideoEncoder are not available. VCE: AMF Error 1. g, NVENC: NvEnc NvEncoderD3D11 failed. Code=15 NvEncoder::LoadNvEncApi : Current Driver Version does not support this NvEncodeAPI version, please upgrade driver at cpp/platform/win32\NvEncoder.cpp:90
16:43:55.226138100 | [INFO] | ClientDisconnected

i am confuse if run ALVR exe and APK from github, it will not meet such problem. 
https://github.com/alvr-org/ALVR/releases/tag/v19.1.0
what's the difference?

does somebody meet this issue and how to solve it?

Run on windows 10 with NVIDIA GeForce RTX20280 Ti 

Many thanks~

**Comment by zmerp** [comment, marked as answer]
Try to update your nvidia driver to latest.

**Reply by panxy2013** [reply]
issue resolve after update the latest NVIDIA driver. thanks~
